packages/e2e-tests/src/specs/editor/backgroundAudio.js

```diff
@@ -64,7 +64,7 @@ describe('Background Audio', () => {
         visible: true,
       });
 
-      const fileName = await uploadFile('audio.mp3');
+      const fileName = await uploadFile('audio.mp3', false);
```

Comment:
removed to test to see if the file check was causing these tests to fail --- this seems to be the case